testsuite/features/reposync/srv_sync_channels.feature

```diff
@@ -25,13 +25,13 @@ Feature: Be able to list available channels and enable them
   Scenario: List products
     When I execute mgr-sync "list products"
     Then I should get "[ ] SUSE Linux Enterprise Server 12 SP4 x86_64"
-    And I should get "[ ] SUSE Manager Proxy 4.1 x86_64"
+    And I should get "[ ] SUSE Manager Proxy 4.2 x86_64"
 
 @susemanager
   Scenario: List all products for SUSE Manager
     When I execute mgr-sync "list products --expand"
     Then I should get "[ ] SUSE Linux Enterprise Server 12 SP4 x86_64"
-    And I should get "[ ] SUSE Manager Proxy 4.1 x86_64"
+    And I should get "[ ] SUSE Manager Proxy 4.2 x86_64"
```

Comment:
We could even use 4.3 isn't it? It's just checking the product is available.

Reply:
We could, yes. OK, let's jump one version.